addons/payment_demo/models/payment_transaction.py

```diff
@@ -78,7 +78,7 @@ def _send_payment_request(self):
         notification_data = {'reference': self.reference, 'simulated_state': simulated_state}
         self._handle_notification_data('demo', notification_data)
 
-    def _send_refund_request(self, **kwargs):
+    def _send_refund_request(self, amount_to_refund=None):
```

Comment:
Or here? It seems like it's fine to pack `amount_to_refund` into the kwargs since we don't need it in this override.

Reply:
Anywhere else the override is like this and here it was just `**kwargs`. So once you install this, you cannot call anymore with `_send_refund_request(123)`.

Reply:
`amount_to_refund` is a kwargs, IMO it makes no sense to call the method with an args instead.

Reply:
It is a positional_or_kwargs, if you want to make it a kwarg only, you need a signature like `_send_refund_request(self, *, amount_to_refund=None)`.

Reply:
Changing the signature like this (by using `kwargs`) removes the ability of IDE to get the correct signature and supported arguments

Reply:
Same here. There are plenty of similar overrides that pack unused kwargs and it's something I believe we should keep doing. It never happened that we had to fix a similar bad call and, if we'll need to one day, we'd fix the call itself, not the override.

Reply:
Why fix when you can prevent?

Reply:
I am open to have a linter that accepts some more cases for private methods - as long as the method does not become too complex. I find it simpler to adapt a few signatures compatible with current usage.